PRS5001-6000/PRS5503Ibnatt.xml

```diff
@@ -44,14 +48,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                  
-                  Ibn aṭ-Ṭayyib
-               </persName>
-               <occupation> theol. </occupation>
+            <person sex="1" sameAs="Q2820894">
+              <persName xml:id="n1" xml:lang="ar">أبو الفرج بن الطيب</persName>
+              <persName corresp="#n1" type="normalized">Abū-l-Faraj ibn aṭ-Ṭayyib</persName>
+               <persName xml:id="n2" type="alt" xml:lang="ar">Ibn aṭ-Ṭayyib</persName>
+               <persName xml:id="n3" type="alt">Abū-l-Faraj ʿAbd Allāh Ibn aṭ‑Ṭayyib al-ʿIrāqī</persName>
+               <persName xml:id="n4" type="alt" xml:lang="gez">ዮሐንስ፡ አፈ፡ ወርቅ፡ ምሥራቃዊ፡</persName>
+               <persName corresp="#n4" type="normalized" xml:lang="egz">Yoḥannǝs ʾAfa Warq
+               Mǝśrāqāwi</persName>
+               <persName corresp="#n4" xml:lang="en">John Chrysostom of the East</persName>
+               <persName xml:id="n5" xml:lang="la">Abulpharagius Abdalla Benattibus</persName>
+               <occupation type="literary">theol.</occupation>
+               <death>1043</death>
             </person>
          </listPerson>
+         <listRelation>
+           <relation name="betmas:formerlyAlsoListedAs" active="PRS5503Ibnatt" passive="PRS1345AbulFa"></relation>
+           <relation name="betmas:formerlyAlsoListedAs" active="PRS5503Ibnatt" passive="PRS1342AbulFa"></relation>
+         </listRelation>
```

Comment:
I would add a relation to specify that he is not 'Abalfarag (PRS1013Abalfara)

Reply:
I can not find an appropriate relation for that, the ones we have, syriaca:different-from and saws:isDifferentTo, are for works only...

Reply:
You are right.. maybe a general dc:relation? on the other hand, it could be more appropriate to add it in the Abalfarag record